tests/test.py

```diff
@@ -0,0 +1,101 @@
+import unittest
+from homework import Rectangle
+import math
+
+
+class TestInvalidRectangleProperties(unittest.TestCase):
+
+    def setUp(self):
+        self.rect = Rectangle
+        self.invalid_args = ((0, 0), (-1, -1), (-1, 0),
+                             (0, -1), (1, 0), (0, 1),
+                             (1, -1), (-1, 1))
+
+    def tearDown(self) -> None:
+        del self.rect
+
+    @unittest.skip("This version of Rectangle class doesn't support assessment of values")
+    def test_invalid_params(self):
+        for args in self.invalid_args:
+            with self.subTest(args=args):
+                self.assertRaises(ValueError, self.rect, *args)
+
+
+class TestValidRectangleProperties(unittest.TestCase):
```

Comment:
why you created different test cases? you test one defined module. it's good practice to pull all tests for one modele to one test Case

Reply:
Fixed.